app/design/adminhtml/default/default/template/TIG/MyParcelBE/sales/order/shipment/create/consignment_options.phtml

```diff
@@ -83,44 +80,20 @@ if(true == $_isPakjegemak){
-                    <option value="letter_box" <?php echo $_disabled;?> <?php echo $type == 2 ? 'selected="selected"' : '';?>><?php echo $this->__('Letter box'); ?></option>
-                    <?php endif; ?>
                     <option value="unstamped" <?php echo $_disabled;?> ><?php echo $this->__('Unpaid'); ?></option>
                 </select>
             </p>
         <div id="myparcel_consignment_options">
+            <p>
+                <label for="tig_myparcel_insured" class="normal">(<?php echo $this->__('Total order value: &euro; %s',$this->getOrderTotal());?>) <?php echo $this->__('Insured up to') . ' ' . TIG_MyParcelBE_Model_Api_MyParcel::INSURED_AMOUNT;?> </label>
+                <input id="tig_myparcel_insured" type="checkbox" value="1" name="tig_myparcel[insured]" <?php echo $_insuredSelected;?> />
+
+            </p>
             <?php if ($this->getDestinationCountry() == 'NL'): ?>
-                <p>
-                    <label for="tig_myparcel_home_address_only" class="normal"><?php echo $this->__('Home address only');?></label>
-                    <input id="tig_myparcel_home_address_only" type="checkbox" value="1" name="tig_myparcel[home_address_only]" <?php echo $_homeSelected;?> />
-                </p>
                 <p>
                     <label for="tig_myparcel_signature_on_receipt" class="normal"><?php echo $this->__('Signature on receipt');?></label>
                     <input id="tig_myparcel_signature_on_receipt" type="checkbox" value="1" name="tig_myparcel[signature_on_receipt]" <?php echo $_signatureSelected;?> />
                 </p>
-                <p>
-                    <label for="tig_myparcel_return_if_no_answer" class="normal"><?php echo $this->__('Return if no answer');?></label>
-                    <input id="tig_myparcel_return_if_no_answer" type="checkbox" value="1" name="tig_myparcel[return_if_no_answer]" <?php echo $_returnNoAnswerSelected;?> />
-                </p>
-                <p>
-                    <label for="tig_myparcel_xl" class="normal"><?php echo $this->__('Large package');?></label>
-                    <input id="tig_myparcel_xl" type="checkbox" value="1" name="tig_myparcel[is_xl]" <?php echo $_isXlSelected;?> />
-                </p>
-                <p>
-                    <label for="tig_myparcel_insured" class="normal">(<?php echo $this->__('Total order value: &euro; %s',$this->getOrderTotal());?>) <?php echo $this->__('Insured up to:');?> </label>
-                    <select id="tig_myparcel_insured_amount" name="tig_myparcel[insured_amount]" >
-                        <option value="50" <?php echo $_insuredUpTo == 50 ? 'selected="selected"' : '';?>>&euro; 50</option>
-                        <option value="250" <?php echo $_insuredUpTo == 250 ? 'selected="selected"' : '';?>>&euro; 250</option>
-                        <option value="500" <?php echo $_insuredUpTo == 500 && $_insuredUpTo < 1000 ? 'selected="selected"' : '';?>>&euro; 500</option>
```

Comment:
Moet de 500 euro verzekering niet beschikbaar blijven?

Reply:
500 euro hoef je niet meer te selecteren, het is by default 500 euro. hiernaast staat ook een checkbox